src/openforms/submissions/logic/actions.py

```diff
@@ -308,6 +309,57 @@ def get_action_log_data(
         }
 
 
+class DMNConfig(TypedDict):
+    plugin_id: str
+    input_mapping: dict[str, str]
+    output_mapping: dict[str, str]
+    definition_id: str
+    definition_version: str
+
+
+@dataclass
+class EvaluateDMNAction(ActionOperation):
+    input_mapping: dict[str, str]
+    output_mapping: dict[str, str]
+    definition_id: str
+    plugin_id: str
+    definition_version: str = ""
+
+    @classmethod
+    def from_action(cls, action: ActionDict) -> "ActionOperation":
+        dmn_config: DMNConfig = action["action"]["value"]
+
+        return cls(**dmn_config)
+
+    def eval(
+        self,
+        context: DataMapping,
+        log: Callable[[JSONValue], None],
```

Comment:
I think this is now gone-ish with the removal of logic rule logging? @Viicos can you tell more?

Reply:
After taking a look at the `submissions.logic` module, I think it can safely be removed. I'll do it in a separate PR once this is merged